test/index.js

```diff
@@ -9,7 +9,9 @@ test('Tests run', (assert) => {
   assert.end();
 });
 
-test('Greet World', (assert) => {
+
+test('Greet World', (assert) => new Promise((resolve) => {
```

Comment:
Oh, I thought you were telling me that ESLint was failing because of:

```js
test('Greet World', (assert) => new Promise((resolve) => {
```

Sorry for the misunderstanding. =)